src/app/component/eco-news/create-news/create-news-cancel/create-news-cancel.component.scss

```diff
@@ -0,0 +1,50 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans&display=swap');
+@import url('https://fonts.googleapis.com/css?family=Montserrat&display=swap');
+
+$title-font: 'Montserrat';
+$main-font: 'Open Sans';
+
+.main-container {
+  max-width: 650px;
+  max-height: 256px;
+  z-index: 2000;
+  font-family: $main-font;
+  padding: 56px;
+
+  .warning-text {
+    font-family: $title-font;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 21px;
+    line-height: 28px;
+    color: #313131;
+    margin-bottom: 40px;
+  }
+
+  .cta-buttons {
+    display: flex;
+
+    .continue-button,
+    .cancel-button {
+      border-radius: 3px;
+      font-style: normal;
```

Comment:
Please use 'secondary-global-button' and 'primary-global-button' styles

Reply:
You can leave only padding: 16px 40px;

Reply:
fixed